nvm.sh

```diff
@@ -2419,17 +2419,21 @@ nvm_auto() {
   fi
 }
 
-NVM_AUTO_MODE='use'
-if nvm_supports_source_options; then
-  while [ $# -ne 0 ]
-  do
-    case "$1" in
-      --install) NVM_AUTO_MODE='install' ;;
-      --no-use) NVM_AUTO_MODE='none' ;;
-    esac
-    shift
-  done
-fi
-nvm_auto "$NVM_AUTO_MODE"
+process_parameters() {
```

Comment:
1) this function should be added to the list that's `unset` when `nvm unload` is called
2) please prefix its name with `nvm_` to avoid any likely collisions with other functions in the user's shell